src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/appsettings.Development.json

```diff
@@ -1,7 +1,7 @@
 {
   ////#if (IndividualLocalAuth)
   //"Local": {
-  //  "Authority": "https://login.microsoftonline.com/",
+  //  "Authority": "https:////login.microsoftonline.com/",
```

Comment:
Could somebody explain this fix? The quadruple-slash syntax is extremely unusual and runs the risk that, in the future, somebody thinks *this* is a bug and "fixes" it back.

I can see that, in `template.json`, we have this:

```
"replaces": "https:////aadB2CInstance.b2clogin.com/",
```

and

```
"replaces": "https:////login.microsoftonline.com/",
```

... so I guess it matches up, but that seems pretty bizarre too. Is it possible that a more understandable fix would be changing `template.json` to match `appsettings.Development.json` instead of the other way around?

If the quadruple-slashes are needed to work around some kind of template system limitation, for example each alternate slash is treated as an escape character for the next slash, then I understand.

Reply:
This is somewhat mixed usage. Some templates are attaching backslash directly into the project template file as we see here. 

https://github.com/dotnet/aspnetcore/blob/e6a5af6633f08890afbd99a6ca7687c623454768/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/appsettings.json#L17 

And for some, it's using a template.json file. 

Both are correct ways, but what is the correct practice or guideline here!?

Reply:
Looking at the code in `template.json`, it replaces `"https:////login.microsoftonline.com/"` (four slash) to `"https://login.microsoftonline.com/"` (two slash). 

But in the file, it was already two slashes only, so the template will now convert `"https://login.microsoftonline.com/"` (two slashes) to `"https:login.microsoftonline.com/"`  (no slash, which is a bug! )

So with this PR, I added two more slashes, so that the template engine will get the exact string to replace in the template as we defined in `template.json`

Reply:
If it's possible to fix this in `template.json` so that we only have normal two-slash URLs in the codebase, that seems like it would be preferable. Do you happen to know if that creates any additional complexity?

(TBH I still don't see how we ended up in the no-slash case, since it's not like the template engine is removing a specific number of slashes. It's just doing string-replacement, and none of the `defaultValue` entries in `template.json` had zero slashes.)

Reply:
After looking at this more, I see that, even though the quadruple-slash syntax is strange, it's actually what we already do in `appsettings.json`. This PR just makes `appsettings.Development.json` consistent with that. As such I think it must be right, unless we were going to do some wider changes that would affect multiple other files.